### ISSUE TITLE
Limit care system edit permissions

### ISSUE THREAD

**Issue by davidmiller** [nhsengland/phmi#20]
On the call 29/10 there was discussion of limiting who is able to add / edit care systems.

Broadly because of a concern that the tool would descend into chaos if we didn't.

Consider options for how to do this without restricting things too much or engendering a high ongoing support overhead.

**Comment by CDampney** [comment]
This is correct in that we need to consider an RBAC approach to editing care systems and reference tables for drop down lists.

However not a priority for this itteration. DO NOT PROCEED return to the inbox.
